### PR TITLE
updated tests for pbs_fairshare.py

### PR DESCRIPTION
**Pull request by anamikau**
#### Bug/feature Description
* *Bugs: test_fairshare_formula5 is failing due to sched_config errors on some platforms*
* *Bugs: All tests are failing due to mismatch in jobs political order*
* *If there is an issue ID, link to it: [PP-1084](https://pbspro.atlassian.net/browse/PP-1084)*

#### Affected Platform(s)
* *Platform : All*

#### Cause / Analysis / Design
* *Bugs: sched_config value was not set as documented hence on some paltforms tests were failing. Also all tests were failing intermittently as all jobs are getting submitted at same time but for some cases it is observed that scheduler sometime picks job3 over job1 causing the tests to fail.*

#### Solution Description
* *Added sleep in between job submission to get different timestamp*

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by anamikau** [comment]
test logs with and without fix. with fix test execution time has increased by ~20s

[withfix.txt](https://github.com/PBSPro/pbspro/files/1729537/withfix.txt)
[withoutfix.txt](https://github.com/PBSPro/pbspro/files/1729538/withoutfix.txt)

File: test/tests/functional/pbs_fairshare.py [hunk 1]
```diff
@@ -157,10 +157,14 @@ def test_fairshare_formula3(self):
         self.server.manager(MGR_CMD_SET, SERVER, {'job_sort_formula': formula})
         J1 = Job(TEST_USER2)
         jid1 = self.server.submit(J1)
+        # Adding sleep for different job submission timestamp
+        time.sleep(1)
```
Comment (bhroam) [inline test/tests/functional/pbs_fairshare.py]: I don't see the need to have different timestamps for the different jobs.  The test just needs to have 4 of them submitted before one cycle.  The test is matching the scheduler log, not anything to do with the job times.

Comment (anamikau) [reply test/tests/functional/pbs_fairshare.py]: you are right. I will remove it.

**Comment by anamikau** [comment]
Hey @vstumpf . No this is a different test case with similar failure as in PR#565.

File: test/tests/functional/pbs_fairshare.py [hunk 2]
```diff
@@ -222,7 +226,7 @@ def test_fairshare_formula5(self):
         """
 
         self.set_up_resource_group()
-        a = {'log_filter': 2048, 'fair_share': "True All"}
+        a = {'log_filter': 2048, 'fair_share': "True ALL"}
```
Comment (vstumpf) [inline test/tests/functional/pbs_fairshare.py]: does this not require double quotes like #565?
or was it rejected because it was `All` instead of `ALL`

Comment (anamikau) [reply test/tests/functional/pbs_fairshare.py]: it doesn't require double quotes. Yes the problem was "All" vs "ALL"
# fair_share
#       Schedule jobs based on usage and share values
#
#       PRIME OPTION
#

fair_share: false       ALL

Comment (vstumpf) [reply test/tests/functional/pbs_fairshare.py]: Thanks

File: test/tests/functional/pbs_fairshare.py (hunk 1 above)
Comment (bhroam) [reply test/tests/functional/pbs_fairshare.py]: There were other sleeps added in other tests.  Are they needed?

Comment (anamikau) [reply test/tests/functional/pbs_fairshare.py]: I see what you are saying. I saw a couple of tests failing due to political order mismatch. Unfortunately I do not have the sched_logs and I am not able to reproduce this now. I will remove the sleep from all the tests.

Comment (bhroam) [reply test/tests/functional/pbs_fairshare.py]: If you want to enforce a political ordering, set the job_sort_formula to 'job_priority' and submit the jobs with the priority you want.  The jobs with the highest priority will be considered first.

**Comment by anamikau** [comment]
@bhroam I have removed the sleep from everywhere. I will monitor the failures if it happen again.

[latest_fairshare.txt](https://github.com/PBSPro/pbspro/files/1749643/latest_fairshare.txt)

**Comment by bhroam** [comment]
Looks good to me.  I sign off.  Please squash and rebase for the merge.

**Comment by anamikau** [comment]
Thanks @bhroam . squashed and rebased. Please take this forward once all checks passes.

**Comment by bhroam** [comment]
I merged in the PR

**Pull request merged**